UKHomeOffice/reference-data-api-service
# Add service health check

To test
* Deploy using docker (please refer to the README);
* Open a new terminal tab and execute `docker ps` over and over until you see the status `healthy` should take around 10secs;
* To test `unhealthy` status, change the status returned from the health route from 200 to 400 and repeat the steps above.
* Console log is required to store the messages from triggered by the Docker `HEALTHCHECK` command, these messages can be seen in the when inspecting the container.
